docs/configuration_options.md

```diff
@@ -2058,6 +2058,24 @@ local docker network to use, if there's need to have db cluster connect to other
 **type:** str (appendable)
 
 
+## **vs_docker_image** / SCT_VS_DOCKER_IMAGE
+
+Vector Store docker image repo
+
+**default:** scylladb/vector-store
+
+**type:** str (appendable)
+
+
+## **vs_version** / SCT_VS_VERSION
```

Comment:
I would unify these into the entire container string, i.e. `scylladb/vector-store:latest`. It is one less configuration option and is consistent with how are containers usually specified